src/service/user.service.js

```diff
@@ -0,0 +1,65 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { User } from '../moduls/User.js';
+import { Console } from '../untils/Console.js';
```

Comment:
There is a typo in the import path: 'untils' should be 'utils'. Please correct the path to '../utils/Console.js'.